File: src/main/java/com/example/demo/user/dto/SignUpForm.java
package com.example.demo.user.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class SignUpForm {
    private String email;
    private String password;
    private String username;
    private String nickname;
    private String imageUrl;
}

File: src/main/kotlin/com/classy/placelog/auth/dto/SignUpForm.kt
package com.classy.placelog.auth.dto

/**
 * 회원가입 폼
 * @property email 이메일
 * @property password 비밀번호
 * @property name 이름
 * @property nickname 닉네임
 */
data class SignUpForm(
    val email: String,
    val password: String,
    val name: String,
    val nickname: String,
)
